src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -56,32 +56,39 @@
     private ValidationStringency validationStringency;
 
     /**
-     * Open CRAM data for reading using either the file or the input stream
-     * supplied in the arguments. The
-     * {@link htsjdk.samtools.Defaults#REFERENCE_FASTA default} reference fasta
-     * file will be used.
+     * Create a CRAMFileReader from either a file or input stream using the {@link ReferenceSource#getDefaultCRAMReferenceSource() default}
+     * reference source.
      *
      * @param cramFile CRAM file to open
-     * @param inputStream   CRAM stream to read
+     * @param inputStream CRAM stream to read
+     *
+     * @throws IllegalArgumentException if the {@code cramFile} and the {@code inputStream} are both null
+     * @throws IllegalStateException if a {@link ReferenceSource#getDefaultCRAMReferenceSource() default}
+     * reference source cannot be acquired
      */
     public CRAMFileReader(final File cramFile, final InputStream inputStream) {
-        this(cramFile, inputStream, new ReferenceSource(Defaults.REFERENCE_FASTA));
+        this(cramFile, inputStream, ReferenceSource.getDefaultCRAMReferenceSource());
```

Comment:
It's also silly and confusing to have a constructor that takes both a `File` and `InputStream` for the same file -- seems like it should be split into two 1-arg constructors (unless `inputStream` is really meant to be for the index).